Simplified/Network/NYPLNetworkQueue.swift

```diff
@@ -84,7 +84,7 @@ final class NetworkQueue: NSObject {
       let methodString = method.rawValue
       let dateCreated = NSKeyedArchiver.archivedData(withRootObject: Date())
       // left for backward compatibility
-      let headerData = NSKeyedArchiver.archivedData(withRootObject: [:])
+      let headerData: Data? = nil
```

Comment:
I looked at the code in the history, since we are no longer sending header data as a parameter (line 78) this is equivalent according to the history.